app/src/organisms/Desktop/Devices/ProtocolRun/SetupRobotCalibration.tsx

```diff
@@ -27,9 +36,33 @@ import { SetupDeckCalibration } from './SetupDeckCalibration'
 import { SetupInstrumentCalibration } from './SetupInstrumentCalibration'
 import { SetupTipLengthCalibration } from './SetupTipLengthCalibration'
 
+import type { AnalyticsProtocolProceedButtonText } from '/app/redux/analytics/constants'
 import type { ProtocolCalibrationStatus } from '/app/redux/calibration/types'
 import type { StepKey } from '/app/redux/protocol-runs'
 
+const determineNextStepButtonKey = (
+  nextStep: StepKey
+): AnalyticsProtocolProceedButtonText => {
+  let nextStepButtonKey: AnalyticsProtocolProceedButtonText
```

Comment:
```suggestion
  let nextStepButtonKey: AnalyticsProtocolProceedButtonText
```
For the most part, we should do our best to avoid `let` bindings and lean on `const`. They lead to unintentional bugs down the development road, as it's too easy to reassign a variable unexpectedly, especially if code becomes more complex.

In this specific situation, let's return the `nextStepButtonKey` directly within each `case`.